reVX/plexos/rev_reeds_plexos.py

```diff
@@ -38,9 +48,13 @@ def __init__(self, sc_build, cf_fpath, cf_res_gids=None,
             resource_gid, and capacity at each SC point.
         cf_fpath : str
             File path to capacity factor file to get profiles from.
-        cf_res_gids : list | np.ndarray, optional
+        res_gids : list | np.ndarray, optional
             Resource GID's available in cf_fpath, if None pull from cf_fpath,
             by default None
+        force_full_build : bool
+            Flag to ensure the full requested buildout is built at each SC
+            point. If True, the remainder of the requested build will always
+            be built at the last resource gid in the sc point.

```

Comment:
Do we have a check for the 1e6 hack? I.e. what if the requested build is > the capacity of the last resource gid? Do we just multiply the cf of the last resource gid by the requested capacity?

Reply:
Yeah this flag turns on the 1e6 hack and i added a negative test with the flag=false to ensure that it will fail if the requested build > available capacity within one watt. If this flag is true, the last GID in the supply curve point is given a 1e6MW capacity ("the remainder of the requested build will always be built at the last resource gid in the sc point")